CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.0.112 (Feb 12, 2024)

```

Comment:
Make sure to update since there we released a few versions since this PR was opened.

Reply:
Done